multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/FlowableConfiguration.java

```diff
@@ -73,22 +75,32 @@ public SpringProcessEngineConfiguration processEngineConfiguration(DataSource da
 
     @Inject
     @Bean
-    public AsyncExecutor jobExecutor(ApplicationConfiguration configuration, String jobExecutorId) {
-        DefaultAsyncJobExecutor jobExecutor = new DefaultAsyncJobExecutor();
-        scale(configuration, jobExecutor);
+    public AsyncExecutor jobExecutor(DefaultAsyncTaskExecutor asyncTaskExecutor, String jobExecutorId) {
+        DefaultAsyncJobExecutor jobExecutor = new DefaultAsyncJobExecutor() {
+            @Override
+            protected void initAsyncJobExecutionThreadPool() {
+                asyncTaskExecutor.start();
+                this.taskExecutor = asyncTaskExecutor;
+                this.shutdownTaskExecutor = true;
```

Comment:
As far as I see the shutdown of the TaskExecutor depends on this variable. Does this variable change over time? Do we need to shutdown the TaskExecutor manually in ApplicationShutdownResource?

Reply:
It is not needed, because the job executor triggers the shutdown, once this variable is set to true.